src/main/resources/application.properties

```diff
@@ -6,6 +6,8 @@ piwik.siteId=${PIWIK_SITE_ID}
 enquiries=mailto:enquiries@companieshouse.gov.uk
 home.path=dev-hub/home
 home.url=
+absoluteHome.url=${REDIRECT_URI}
+historicHome.url=api/docs
```

Comment:
Should this have a preceding slash `/api...`?

Reply:
Doesn't seem to need it but it would be more consistent.